examples/jsm/nodes/accessors/LineMaterialNode.js

```diff
@@ -15,11 +15,15 @@ class LineMaterialNode extends MaterialNode {
 LineMaterialNode.SCALE = 'scale';
 LineMaterialNode.DASH_SIZE = 'dashSize';
 LineMaterialNode.GAP_SIZE = 'gapSize';
+LineMaterialNode.LINEWIDTH = 'linewidth';
+LineMaterialNode.DASH_OFFSET = 'dashOffset';
 
 export default LineMaterialNode;
 
 export const materialLineScale = nodeImmutable( LineMaterialNode, LineMaterialNode.SCALE );
+export const materialLineDashOffset = nodeImmutable( LineMaterialNode, LineMaterialNode.DASH_OFFSET );
 export const materialLineDashSize = nodeImmutable( LineMaterialNode, LineMaterialNode.DASH_SIZE );
 export const materialLineGapSize = nodeImmutable( LineMaterialNode, LineMaterialNode.GAP_SIZE );
+export const materialLineWidth = nodeImmutable( LineMaterialNode, LineMaterialNode.LINEWIDTH );
```

Comment:
Could you please add these new exports to Nodes.js?

Reply:
done